### PR TITLE
Add Python solution to CSES 1074 (Stick Lengths)

### PR DESCRIPTION
**Pull request by jaredliw**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as
done._

- [x] I have tested my code
- [x] I have followed the code style guidelines mentioned
      [here](https://usaco.guide/general/adding-solution/#code-conventions)
- [x] I have properly formatted the files according to the steps mentioned
      [here](https://usaco.guide/general/adding-solution#steps)

**The naming of variables inherits from C++ code.**

### CODE REVIEW

File: solutions/cses-1074.mdx [hunk 1]
```diff
@@ -89,4 +89,19 @@ public class StickLengths {
 
 </JavaSection>
 
+<PySection>
+
+```python
+n = int(input())
+p = list(map(int, input().split()))
+p.sort()
```
Comment (SansPapyrus683) [inline solutions/cses-1074.mdx:97]:
```suggestion
p = sorted(list(map(int, input().split())))
```

File: solutions/cses-1074.mdx [hunk 2]
```diff
@@ -89,4 +89,18 @@ public class StickLengths {
 
 </JavaSection>
 
+<PySection>
+
+```python
+n = int(input())
+p = sorted(list(map(int, input().split())))
```
Comment (SansPapyrus683) [inline solutions/cses-1074.mdx:96]: (optional) rename to `sticks`

**Review by SansPapyrus683 — APPROVED** [review]
bing chilling

File: solutions/cses-1074.mdx (hunk 2 above)
Comment (jaredliw) [reply solutions/cses-1074.mdx]: alright

**Pull request merged**